briskedu/brisk-learning
# Add Sapling integration

Summary:
Adding in Sapling, though their trial API is more limited than GPTZero's.

https://app.asana.com/0/1203963408726580/1204124172035073

Test Plan:
1. Run tests
2. Confirm extension works
